code_enforcement_case.go

```diff
@@ -25,6 +25,9 @@ type CodeEnforcementCases struct {
 		CaseSource               string      `xml:"case_source"`
 		Description              string      `xml:"description"`
 		OpenDate                 string      `xml:"open_date"`
+		CloseDate                string      `xml:"close_date"`
```

Comment:
what format is this date in? I would like to parse it into a time.Time struct so we get access to all the time package methods.